orchestration/src/main/java/com/sap/ai/sdk/orchestration/client/model/SearchDocumentKeyValueListPair.java

```diff
@@ -12,93 +12,68 @@
 
 package com.sap.ai.sdk.orchestration.client.model;
 
-import com.fasterxml.jackson.annotation.JsonAnyGetter;
-import com.fasterxml.jackson.annotation.JsonAnySetter;
-import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.LinkedHashMap;
 import java.util.List;
-import java.util.Map;
-import java.util.NoSuchElementException;
 import java.util.Objects;
-import java.util.Set;
-import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
 
 /** SearchDocumentKeyValueListPair */
-// CHECKSTYLE:OFF
-public class SearchDocumentKeyValueListPair
-// CHECKSTYLE:ON
-{
-  @JsonProperty("key")
+@JsonPropertyOrder({
+  SearchDocumentKeyValueListPair.JSON_PROPERTY_KEY,
+  SearchDocumentKeyValueListPair.JSON_PROPERTY_VALUE,
+  SearchDocumentKeyValueListPair.JSON_PROPERTY_SELECT_MODE
+})
+@javax.annotation.Generated(
+    value = "org.openapitools.codegen.languages.JavaClientCodegen",
+    comments = "Generator version: 7.9.0")
+public class SearchDocumentKeyValueListPair {
```

Comment:
(Major)
Looks like some classes do not have the `@Beta` annotation(?)

Reply:
This comes from not deleting the output directory before generating. To ensure we don't have any dead code, we should enable this. If it's not possible to use the OSS generator flag, we can use the maven clean plugin (refer to [this code](https://github.com/SAP/ai-sdk-java/commit/b7d7f6e8f8ba784aec0a6e0f5f9c72b8a1883ceb#diff-8d04401f1cc51365fe3e32f019cd720135ba920a1a7da7f19e9c9208478701fcL148)).

Reply:
I see we currently have some manually maintained classes in the package. Maybe we can move the manually maintained classes out such that we can safely delete and re-generate the package?

Reply:
> Maybe we can move the manually maintained classes out

Or further leverage the `mixIn` logic to override annotations.

Reply:
Done.